mailpoet/lib/Config/Menu.php

```diff
@@ -412,6 +413,23 @@ public function setup() {
         'mailpoet-automation',
         [$this, 'automation']
       );
+
+      // Automation editor
+      $automationEditorPage = $this->wp->addSubmenuPage(
+        true,
+        $this->setPageTitle('Automation Editor'),
+        'Automation Editor',
+        AccessControl::PERMISSION_MANAGE_EMAILS,
```

Comment:
Afaik the default `capability` for the rest api endpoint is `administrator` while here it is `mailpoet_manage_emails` (defaults to admin and editor). I think this needs to be streamlined. https://github.com/mailpoet/mailpoet/blob/master/mailpoet/lib/Automation/Engine/API/Endpoint.php#L12-L14

Reply:
👍 Addressed in https://github.com/mailpoet/mailpoet/pull/4131/commits/7dc50e1454785c97ac033182cbb91f0909829dfe. For now I just used MailPoet's AccessControl class and noted we may extract it in the future.